build/package-lists/java/package-list

```diff
@@ -170,8 +170,8 @@ javax.xml.transform.stax
 javax.xml.transform.stream
 javax.xml.validation
 javax.xml.ws
-javax.xml.ws.handler
-javax.xml.ws.handler.soap
+javax.xml.ws.desiredAction
+javax.xml.ws.desiredAction.soap
```

Comment:
What's this change about?

Reply:
This commit fixes a Javadoc issue, but I've taken the opportunity to rename this enum to (what I think is) a more descriptive name.